scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -6,22 +6,31 @@ include::_attributes/common-attributes.adoc[]
+Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
 
-The minimum number of reserved CPUs required for the management partition for a single-node cluster is four CPU Hyper threads (HTs). The set of pods that make up the baseline {product-title} installation and a set of typical add-on Operators are annotated for inclusion in the management workload partition. These pods operate normally within the minimum size `cpuset` configuration. Inclusion of Operators or workloads outside of the set of accepted management pods requires additional CPU HTs to be added to that partition.
+Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
 
-Workload partitioning isolates the user workloads away from the platform workloads using the normal scheduling capabilities of Kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing cluster management workloads and user workloads.
+The following is an overview of the configurations required for workload partitioning:
 
-When applying workload partitioning, use the Node Tuning Operator to implement the performance profile:
+* Workload partitioning that uses `/etc/crio/crio.conf.d/01-workload-partitioning` pins the {product-title} infrastructure pods to a defined `cpuset` configuration.
 
-* Workload partitioning pins the {product-title} infrastructure pods to a defined `cpuset` configuration.
-* The performance profile pins the systemd services to a defined `cpuset` configuration.
-* This `cpuset` configuration must match.
+* The performance profile pins cluster services such as systemd and kubelet to the CPUs that are defined in the `spec.cpu.reserved` field.
++
+[NOTE]
+====
+Use the Node Tuning Operator to implement the performance profile to also pin system level apps for a complete workload partitioning configuration on the node.
+====
 
-Workload partitioning introduces a new extended resource of `<workload-type>.workload.openshift.io/cores`
-for each defined CPU pool, or workload-type. Kubelet advertises these new resources and CPU requests by pods allocated to the pool are accounted for within the corresponding resource rather than the typical `cpu` resource. When workload partitioning is enabled, the `<workload-type>.workload.openshift.io/cores` resource allows access to the CPU capacity of the host, not just the default CPU pool.
+* The CPUs that you specify in the performance profile `spec.cpu.reserved` field and the workload partitioning `cpuset` field must match.
+
+Workload partitioning introduces an extended `<workload-type>.workload.openshift.io/cores` resource for each defined CPU pool, or workload-type.
+Kubelet advertises the resources and CPU requests by pods allocated to the pool within the corresponding resource rather than the typical `cpu` resource.
```

Comment:
```suggestion
Kubelet advertises the resources and CPU requests by pods allocated to the pool within the corresponding resource rather than within the typical `cpu` resource.
```

Reply:
suggestion above for clarity but may not be needed